src/components/Menu/Menu.test.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-multi-assign */
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import Menu from './index';
+
+import { ConfigProvider } from '../../context/State/state';
+
+const valueContextMock = {
+  darkMode: false,
+  video: {
+    openedDetails: false,
+  },
+};
+
+beforeEach(() => {
+  render(
+    <ConfigProvider value={valueContextMock}>
+      <Menu />
+    </ConfigProvider>
+  );
+});
+
+describe('Menu', () => {
+  test('should render Home span', async () => {
+    const spanLabel = screen.queryByText('HOME');
```

Comment:
```suggestion
    const spanLabel = screen.getByText('HOME');
```
`query*` should be used only when we check for inexistent. See: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence